site/content/docs/seo.md

```diff
@@ -0,0 +1,39 @@
+---
+title: SEO Configuration
+description: How to set up SEO in flowershow
+---
+
```

Comment:
Brief explanation of what we mean by SEO: e.g. 

Flowershow supports adding standard metadata to your pages to improve social media sharing and search engine discovery.

As well as setting values for individual pages you can add a default image and description that will be used on pages without this set.